src/tests/JIT/HardwareIntrinsics/X86/Avx512BW/Store.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics;
+using Xunit;
+
+namespace IntelHardwareIntrinsicTest._Avx512BW
+{
+    public partial class Program
+    {
+        [Fact]
+        public static unsafe void Store()
+        {
+            int testResult = Pass;
+
+            if (Avx512BW.IsSupported)
+            {
+                using (TestTable<short> intTable = new TestTable<short>(new short[32] { 1, -5, 100, 0, 1, 2, 3, 4, 1, -5, 100, 0, 1, 2, 3, 4, 1, -5, 100, 0, 1, 2, 3, 4, 1, -5, 100, 0, 1, 2, 3, 4 }, new short[32]))
```

Comment:
```suggestion
                using (TestTable<short> intTable = new(new short[32] { 1, -5, 100, 0, 1, 2, 3, 4, 1, -5, 100, 0, 1, 2, 3, 4, 1, -5, 100, 0, 1, 2, 3, 4, 1, -5, 100, 0, 1, 2, 3, 4 }, new short[32]))
```

a bit more terse; here and elsewhere

Reply:
This is setup in a way that makes it easier to switch to generated from a template in the future.